fastar/rules.py

```diff
@@ -264,25 +288,32 @@ def conv_incounts(lhs_shape, rhs_shape, window_strides):
               for lo, hi, tile_count in zip(lo, hi, tile_counts)]
     single_dim_counts.append(np.sum(counts, axis=0))
   lhs_count = np.broadcast_to(
-    outer_product(single_dim_counts) * outchannel_size, lhs_shape)
-  return lhs_count, np.full(rhs_shape, batch_size)
+    outer_product(single_dim_counts) * outchan, lhs_shape)
+  return lhs_count, None
```

Comment:
Why is `rhs_count` not `np.full(rhs_shape, batch_size)`?

Reply:
Shouldn't it be `np.full(rhs_shape, batch_size * out_chan)`? I guess I could just correct it :slightly_smiling_face:.

Reply:
There is a separate slice in the kernel per output dimension. Therefore, it should be independent of outchan.

Reply:
Very good point, I'll un-remove this stuff.